drakrun/drakrun/playground.py

```diff
@@ -87,10 +87,25 @@ def drakvuf(self, plugins, timeout=60):
 
         return d
 
+    def help(self):
+        usage = """
+        Available commands:
+        - copy(file_path)   # copy file onto vm desktop
+        - mount(iso_path)   # mount iso, useful for installing software, e.g. office
+        - drakvuf(plugins)  # start drakvuf with provided set of plugins
+        - run(cmd)          # run command inside vm
+        - exit()            # exit playground, remember to run postinstall if you want to save vm state
```

Comment:
>remember to run postinstall if you want to save vm state

I'm not sure if we want to put this here :thinking: This may be misleading if someone runs e.g. `drakplayground 1`.
This would only work if we're running vm-0

Reply:
Hmm, in that case I think it would be a good idea to add additional warning `prompt` when user exits drakplayground 0. I will create a separate pr. for this later.